replace.json

```diff
@@ -0,0 +1,3 @@
+{
+  "/* @echo FONT_CDN_PATH *":"https://dhrlmnmyf2njb.cloudfront.net"
```

Comment:
This should really be an environment variable rather than committing it into source control. We're using different values for staging and production which this method then supports.

Anything that has a reference to a path or an ID it's often useful to try and store them as a variable that can be configured differently depending on the environment the app is being run on.